examples/chatbot/README.md

```diff
@@ -1,23 +1,39 @@
 # Retrieval-based Chatbot
 
-This example showcases the use of `Forte` to build a retrieval-based chatbot and perform text 
-analysis on the retrieved results. We use the dataset released as part of this paper 
-[Target-Guided Open-Domain Conversation](https://arxiv.org/abs/1905.11553). The dataset consists 
-of conversations between two entities A and B. We finetune a BERT model that helps retrieve a 
+This example showcases the use of `Forte` to build a retrieval-based chatbot and perform text
+analysis on the retrieved results. We use the dataset released as part of this paper
+[Target-Guided Open-Domain Conversation](https://arxiv.org/abs/1905.11553). The dataset consists
+of conversations between two entities A and B. We finetune a BERT model that helps retrieve a
 response for a context.
 
 **Note**: All the commands below should be run from `examples/chatbot/` directory.
 
-In this example, the user speaks in German and the bot extracts information stored in English. The 
-bot finally translates the response to German. For text analysis, we run a *Semantic Role 
-Labeler* (SRL) on the retrieved response to identify predicate mentions and arguments. Let us see a 
+In this example, the user speaks in German and the bot extracts information stored in English. The
+bot finally translates the response to German. For text analysis, we run a *Semantic Role
+Labeler* (SRL) on the retrieved response to identify predicate mentions and arguments. Let us see a
 step-by-step guide on how to run this example.
 
+## Packages
+After setting up `forte` environment by installing packages in `requirements.txt` in the project root folder.
+```
+pip install termcolor
+pip install tensorflow==2.4 # version compantible with numpy==1.19.5
+pip install faiss-gpu
+```
+
+Then we need to install forte external pacakges
+```
+pip install src/faiss
```

Comment:
This instruction is not clear, since it didn't specify the repo you need to download and the directory you are in.

Reply:
changed the description.